Game/Source/GDKTestGyms/AUptimeGameMode.cpp

```diff
@@ -20,7 +20,7 @@ DEFINE_LOG_CATEGORY(LogUptimeGymGameMode);
 
 namespace
 {
-	const FString PlayerDensityWorkerFlag = TEXT("player_density");
+	const FString PlayerDensityWorkerFlag1 = TEXT("player_density");
```

Comment:
I suspect the right solution to this namespace collision would be to just give the anonymous namespace a name, though this depends on what the rest of the team decides on this issue.

Reply:
Added a namespace

Reply:
Giving it a name removes the benefits of it being anonymous in the first place. The GDK team is looking at how best to deal with these, so now that the variable name conflict has been resolved, I'd keep it nameless.

Reply:
Going by the slack discussion and vote it sounded to me like we decided on not using anonymous namespaces.